src/main/java/frc/robot/Constants.java

```diff
@@ -56,11 +65,45 @@ public static class CANIDs {
   }
 
   public static class DIOConstants {
-    public static final int kthroughBoreEncoderID = 1;
+    public static final int kthroughBoreEncoderID = 10;
```

Comment:
Need this to be reverted for real robot. 
Put placeholders (with FIXME comments) for the climber encoders